uwblueprint/supportive-housing
# Log Record Row Expansion

## Notion task link


[Task Name](https://www.notion.so/uwblueprintexecs/Log-Record-Row-Expansion-a2bfd26e2d484669b23a187e76005436)



## Implementation description
* Created a new option to "view" log records
* Design is similar to the edit modal, except all inputs are "disabled"
* Clicking the edit button in the view modal brings up the edit modal



## Steps to test
1. Create a log record with building, residents, tags, etc.
2. Click the three dots next to the log record and press the "View log record" button
3. A view modal should pop up and ensure that all data displayed is correct and accurate
4. Make sure that you can't modify any data at this stage
5. Click the edit button to bring up the edit modal
6. You can also click the close button to make sure the view modal closes



## What should reviewers focus on?
* Let me know if the design should be retweaked or if the implementation could be improved!


## Checklist
